queue/src/fiskaltrust.Middleware.Localization.QueueIT/v2/DailyOperations/ZeroReceipt0x200.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
+using fiskaltrust.Middleware.Contracts.Extensions;
+using fiskaltrust.Middleware.Contracts.Repositories;
+using fiskaltrust.Middleware.Localization.QueueIT.Constants;
+using fiskaltrust.storage.V0;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT.v2.DailyOperations
+{
+    public class ZeroReceipt0x200 : IReceiptTypeProcessor
+    {
+        private readonly IITSSCD _itSSCD;
+        private readonly ILogger<ZeroReceipt0x200> _logger;
+        private readonly IConfigurationRepository _configurationRepository;
+        private readonly IMiddlewareQueueItemRepository _middlewareQueueItemRepository;
+
+        public ITReceiptCases ReceiptCase => ITReceiptCases.ZeroReceipt0x200;
+
+        public ZeroReceipt0x200(IITSSCD itSSCD, ILogger<ZeroReceipt0x200> logger, IConfigurationRepository configurationRepository, IMiddlewareQueueItemRepository middlewareQueueItemRepository)
+        {
+            _itSSCD = itSSCD;
+            _logger = logger;
+            _configurationRepository = configurationRepository;
+            _middlewareQueueItemRepository = middlewareQueueItemRepository;
+        }
+
+        public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ExecuteAsync(ftQueue queue, ftQueueIT queueIT, ReceiptRequest request, ReceiptResponse receiptResponse, ftQueueItem queueItem)
+        {
+            var signingAvailable = false;
+            try
+            {
+                var deviceInfo = await _itSSCD.GetRTInfoAsync().ConfigureAwait(false);
+                _logger.LogDebug(JsonConvert.SerializeObject(deviceInfo));
+                signingAvailable = true;
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex, "Error on DeviceInfo Request.");
+            }
+
+            if (queueIT.SSCDFailCount == 0)
+            {
+                var log = "Queue has no failed receipts.";
+                if (!signingAvailable)
+                {
+                    log = $"Signing not available. {log}";
```

Comment:
I think we should return state 0x0002 scu temporary out of service if signing is not available ...